packages/trader/src/App/Containers/Modals/success-dialog.jsx

```diff
@@ -24,7 +24,7 @@ const SuccessDialog = ({
     toggleModal,
     title,
     has_close_icon,
-    width = undefined,
+    width = '',
```

Comment:
here I'm avoiding a TS error saying that 'Type 'string' is not assignable to type 'undefined'' because of the module's type alias defining this width as undefined only.